tyxml-ppx-jsx.opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+version: "dev"
+maintainer: "dev@ocsigen.org"
+author: "The ocsigen team"
+homepage: "https://github.com/ocsigen/tyxml/"
+bug-reports: "https://github.com/ocsigen/tyxml/issues"
+doc: "https://ocsigen.org/tyxml/manual/"
+dev-repo: "git+https://github.com/ocsigen/tyxml.git"
+license: "LGPL-2.1 with OCaml linking exception"
+
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+
+depends: [
+  "ocaml" {>= "4.02"}
+  "dune" {build}
+  "alcotest" {with-test}
+  "tyxml"
+  "tyxml-ppx"
```

Comment:
Does it really need the tyxml-ppx dependency too ?

Reply:
In ideal, no. But opam/dune will not install the common lib for jsx ppx because it is a subpackage of tyxml-ppx. Maybe it is better to move the ppx shared lib into tyxml itself ?